examples/map/map_segment.py

```diff
@@ -0,0 +1,98 @@
+"""
+=======================================================
+Segmenting a Map based on transformation of coordinates
+=======================================================
+This example demonstrates extracting a region of a particular map based on
+world coordinates in different systems.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+
+import astropy.units as u
+from astropy.coordinates import SkyCoord
+
+import sunpy.map
+from sunpy.coordinates import NorthOffsetFrame
+from sunpy.data.sample import AIA_171_IMAGE
+
+######################################################################
+# We start with the sample data.
+smap = sunpy.map.Map(AIA_171_IMAGE)
+
+######################################################################
+# A utility function gives us access to the helioprojective coordinate of each
+# pixel. From this we then transform the coordinates to the required frame.
```

Comment:
While this function does give us the helioprojective coordinates for this map, as that is it's native coordinate system, it's not universally true. So just to be pedantic:

```suggestion
# A utility function gives us access to the helioprojective coordinate of each
# pixel in this map. From this we then transform the coordinates to the required frame.
```